judo-runtime-core-dispatcher/src/main/java/hu/blackbelt/judo/runtime/core/dispatcher/VariableResolverManager.java

```diff
@@ -0,0 +1,17 @@
+package hu.blackbelt.judo.runtime.core.dispatcher;
+
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+public interface VariableResolverManager {
+
+    void registerSupplier(final String category, final String key, @SuppressWarnings("rawtypes") final Supplier supplier, final boolean cacheable);
+
+    @SuppressWarnings("unchecked")
+    void registerFunction(final String category, @SuppressWarnings("rawtypes") final Function function, final boolean cacheable);
```

Comment:
I'm not really sure about these SuppressWarnings-s

Reply:
You are right. Removing